src/nap.js

```diff
@@ -225,7 +227,7 @@ function newWeb(){
 
   web.use = function() {
     if(!arguments.length) return web
-    middleware = toArray(arguments).reverse().concat(middleware)
+    middleware.unshift.apply(middleware, toArray(arguments).reverse())
```

Comment:
I don't quite understand the difference here, other than not losing the reference to `middleware` – is that the point? Are we copying references to `middleware` such that they get stale whenever `use` is called?